index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portifolio skeleton</title>
+  <link rel="stylesheet" href="index.css" />
+</head>
+
+<body>
+  <div class="font">
+    <section class="first-section">
+      <div class="first-container">
+        <div class="font">
+          <header class="header">
+            <a class="my-logo">My Logo</a>
+            <nav class="hamburger">
+              <ul class="ul-style">
+                <li>
+                  <a href="#">
+                    <img src="./images/Union.svg" alt="union background" />
+                  </a>
+                </li>
+              </ul>
+            </nav>
+          </header>
+
+          <section class="section1">
+            <h2 class="first-para">
+              I’m Mirela<br />
+              Glad to see <br />
+              you!
+            </h2>
+            <h2 class="second-para">
+              I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded,
+              don’t hestiate to contact me.
+            </h2>
```

Comment:
- [ ] The paragraph should be a `p ` tag. This is part of semantic HTML as your tags should be meaningful according to what is their content. Let's fix that